ledger/ledger-on-memory/src/main/scala/com/daml/ledger/on/memory/InMemoryLedgerReaderWriter.scala

```diff
@@ -140,7 +140,11 @@ object InMemoryLedgerReaderWriter {
   def owner(
       ledgerId: LedgerId,
       participantId: ParticipantId,
-  )(implicit executionContext: ExecutionContext): ResourceOwner[InMemoryLedgerReaderWriter] =
+      clock: Clock = Clock.systemUTC(),
```

Comment:
Nice. :slightly_smiling_face: Would a `() => Instant` work though?

Reply:
Probably, but I find `Clock` quite a nice interface. Any reason you'd prefer the lambda?